modules/ROOT/pages/publish-apis.adoc

```diff
@@ -4,70 +4,114 @@ include::_attributes.adoc[]
+Click the *Apply* button at the bottom right.
 
-If a developer publishes a new API asset version by updating the markdown pages in Exchange, or updating the API specification in API Designer, then ACM imports those changes automatically and immediately.
+Consumer developers now see this API version in the API carousel on the community site.
 
-If a developer publishes a new API major version, it will only be published on ACM when the ACM community administrator publishes it manually.
+== Minor API Updates
 
-// Missing so far: Grouping APIs into API Products
+When an API specification or its documentation need corrections such as changing the text, these non-breaking changes are given a minor version change, such as changing version `3.0.0` to version `3.0.1`. ACM automatically and immediately imports these updates to all communities where that API is published, without requiring any action from the ACM administrator.
+
+For example, suppose both the text in an API's documentation and the text in the API Specification description have spelling errors.
+
+To correct errors in the API specification, open Design Center, open the API project, and fix the spelling in the description.
+
+Click the *Publish to Exchange* button at the top right.
+
+In the dialog box *Publish API specification to Exchange*, leave the existing *Name*, *Main file*, and *API Version*, and increment the *Asset version* for a non-breaking change. For example, if the *Asset version* is `3.0.0`, change it to `3.0.1`.
+
+Click the *Publish* button at the bottom right.
+
+To correct errors in the API documentation, open Exchange, open the API, click the *Edit* button at the top right, and fix the errors.
+
+Click the *Save as draft* button at the bottom right.
+
+Click the *Publish* button at the top right.
+
+Consumer developers now see the corrected information on the community site.
+
+== New API Versions
+
+When a new version of the API with breaking changes is created, the ACM community administrator controls when and how to publish the new version in the community, mark the old version as deprecated, and remove the old version.
+
+Suppose a developer on the API team adds a new resource to an API.
+
+In Design Center the developer opens the API project, adds the resource to the specification, and clicks the *Publish to Exchange* button at the top right.
+
+This is a breaking change, so in the dialog box *Publish API specification to Exchange* the developer changes the *API version*. For example, if the *API version* is `v3`, it becomes `v4`. If the *Asset version* is `3.0.1`, it becomes `4.0.0`. *Name* and *Main file* do not change.
+
+The developer clicks the *Publish* button at the bottom right.
+
+The previous API version 3 remains in all communities where it was published without any changes.
```

Comment:
I think version 3 should be version 3.x or something like that. You have above v3 and then 3.0.1. So version 3 introduces an inconsistency in convention.

Reply:
True. I changed this to "v3" to match the "API version" setting in Design Center.